resources/assets/less/livestreams.less

```diff
@@ -0,0 +1,191 @@
+.livestream-header {
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-image: url("/images/headers/live-top.png");
+	height: 180px;
+	&__container {
+		display: flex;
+		justify-content: flex-start;
+		flex-wrap: wrap;
+		align-items: center;
+		height: 100%;
+		@media @desktop {
+			padding-left: 115px;
+			padding-right: 115px;
+		}
+		padding-left: 10px;
+		padding-right: 10px;
+	}
+	&__headline {
+		color: white;
+		margin: 0;
+		margin-bottom: 10px;
+	}
+	&__description {
+		color: white;
+	}
+}
+.livestream-page {
+	@media @desktop {
+		padding-left: 10% !important;
+		padding-right: 10% !important;
+	}
+	&__header {
+		color: @grey-5;
+		padding-bottom: 15px;
+	}
+	&__anchor-twitch,
+	&__anchor-twitch:link, 
+	&__anchor-twitch:visited, 
+	&__anchor-twitch:hover, 
+	&__anchor-twitch:active {
+		color: inherit;
+	    text-decoration: inherit;
+	    font-weight: inherit;
+	}
+}
+.livestream-featured {

```

Comment:
Yeah, but their structure is different, how do you want me to make it variation if there are more divs in regular than featured?